### DISCUSSION TITLE
Which data are a must-have to develop a new plant model for ApsimX?

### DISCUSSION THREAD

**Discussion by LenLon** [APSIMInitiative/ApsimX#9499] [General]
Hello, more of a general question today. 

We want to develop plant models for some herbs (e.g. cilantro) to use as an undersown crop in an intercropping setup. Field experiments are running, but it would be great to know what are must-haves to calibrate and develop a plant model in ApsimX using its PMF? Or does this to a large extent depend on the intended use of the model (e.g. root development, competition, N uptake, ..)?

Specifications on quality (what to measure) and quantity (how many measurements per season, how many seasons, replicates, ...) are most welcome!

I also checked out the SCRUM Crop Instance model, so data to calibrate that I assume is a good start.

**Comment by HamishBrownPFR** [comment]
Hello @LenLon.
There is a new version of SCRUM which has been developed with the aim of being easy to parameterize for less common crops that do not have much data available.  The main thing to keep in mind with SCRUM is that potential yield is a user input then SCRUM will work out if any stress reduce yield and predict N and water uptake.  It also has a very basic phenology which takes degree days from establishment to harvest or harvest date as an input.  It is good for plugging gaps in rotations or doing competition modeling where a basic plant model is needed. 
The new version has been approved for release but I have not yet moved it into general release.   This will happen soon
The most important data for parameterizing a SCRUM crop is Harvest index, the moisture content of harvested parts and the nitrogen content of the harvested parts and stover (bits that are not harvested).  There is a video of the introduction to SCRUM given to the reference panel which needs to go up on the web site.  I will try to get that sorted too.

**Reply by LenLon** [reply]
Hi Hamish, thanks for the answer!
Great to hear that getting a new release, and glad that it's able to do competition modeling too, as our system is purely intercrop. As we have data on actual yields of the herbs it should be at least possible to determine a potential yield. Can you ping me when the new SCRUM has made it to release? Best, Lennart
